src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationTestService.java

```diff
@@ -75,6 +75,8 @@
 @Service
 class ProgrammingExerciseIntegrationTestService {
 
+    private static final String NON_EXISTING_ID = Integer.toString(1_000_000);
+
```

Comment:
```suggestion

    private static final String NON_EXISTING_ID = Integer.toString(Integer.MAX_VALUE);

```
Why not use Integer.MAX_VALUE here? If the IDs are auto-generated, that should be even more future-proof.